src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Hey CH.

Stephen gave a thumbs up for a topic branch so more people can
see the ticket discussions:

    https://github.com/ga4gh/schemas/tree/sequence-annotation

So please just put in FIXMEs for anything you don't have an
done and commit.  I will then pull in into the topic branch.

I want to send out an e-mail tonight so people have a chance to
look at the branch before tomorrow morning's call

Thanks
Mark